packages/default/scss/wizard/_variables.scss

```diff
@@ -1,14 +1,13 @@
 // Wizard
-$wizard-padding-x: $padding-x * 2.5 !default;
+$wizard-padding-x: $kendo-padding-md-x * 2.5 !default;
 $wizard-padding-y: $wizard-padding-x !default;
-$wizard-content-padding-x: $padding-x * 4 !default;
-$wizard-content-padding-y: $padding-y * 8 !default;
+$wizard-content-padding-x: $kendo-padding-md-x * 4 !default;
+$wizard-content-padding-y: $kendo-padding-md-y * 8 !default;
 $wizard-buttons-margin-x: 10px !default;
 
 $wizard-border-width: 0px !default;
 $wizard-font-size: $font-size !default;
 $wizard-line-height: $line-height !default;
 $wizard-font-family: $font-family !default;
 
-$wizard-step-border-focused: #656565 !default;
-$wizard-focused-shadow: inset 0 0 0 2px rgba(0, 0, 0, .13) !default;
```

Comment:
is the box-shadow var removed by accident here ?

Reply:
it was not used